src/components/sbb-link/sbb-link.tsx

```diff
@@ -14,46 +15,53 @@ import { i18nTargetOpensInNewWindow } from '../../global/i18n';
   styleUrl: 'sbb-link.scss',
   tag: 'sbb-link',
 })
-export class SbbLink {
+export class SbbLink implements AccessibilityProperties {
   /**
    * If set to true, the browser will
-   * show the download dialog on click.
+   * show the download dialog on click (optional).
    */
   @Prop() public download?: boolean;
 
-  /** The href value you want to link to */
-  @Prop() public hrefValue!: string;
+  /** The href value you want to link to (if its not present link becomes a button)*/
+  @Prop() public href?: string;
 
   /**
    * The icon name we want to use,
    * choose from the small icon variants from
    * the ui-icons category from here
-   * https://lyne.sbb.ch/tokens/icons/.
+   * https://lyne.sbb.ch/tokens/icons/ (optional).
    * Inline variant doesn't support icons.
    */
   @Prop() public icon?: string;
 
   /**
    * Pass in an id, if you need to identify
-   * the link element.
+   * the link element (optional).
    */
   @Prop() public idValue?: string;
 
   /**
    * Decide whether the icon should get flipped
-   * horizontally if the document writing mode
-   * is changed from ltr to rtl or vice versa.
+   * horizontally to the left or to the right.
    */
   @Prop() public iconFlip?: boolean;
```

Comment:
I think we have a misunderstanding regarding the iconFlip property. Can it be that you are mix the properties iconPlacement and iconFlip? Icon flip does mirror an icon which for me still only reall makes sense in rtl mode. But I am unsure if we even want to support this edge case in the sbb-link component. @kyubisation your opinion?

Reply:
@HendrikExtSBB  @kyubisation and I decided to remove the iconFlip property, so you can delete it and remove the corresponding css classes.